resources/Nova/js/helpers.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * Helpers for the nova vue components
+**/
+
+export function makeRandomKey(extra = '') {
```

Comment:
Is this ever used anywhere? If it's cruft, let's go ahead and remove it! :)

Reply:
will do, I thought i might need it